test/coreneuron/test_direct.py

```diff
@@ -40,19 +40,35 @@ def test_direct_memory_transfer():
     from neuron import coreneuron
 
     coreneuron.enable = True
+    coreneuron.verbose = 0
     coreneuron.gpu = enable_gpu
 
     pc = h.ParallelContext()
-    h.stdinit()
-    pc.psolve(h.tstop)
-    tran = [h.t, h.soma(0.5).v, h.soma(0.5).hh.m]
 
-    assert tv.eq(tvstd)
-    assert (
-        v.cl().sub(vstd).abs().max() < 1e-10
-    )  # usually v == vstd, some compilers might give slightly different results
-    assert i_mem.cl().sub(i_memstd).abs().max() < 1e-10
-    assert h.Vector(tran_std).sub(h.Vector(tran)).abs().max() < 1e-10
+    def run(mode):
+        h.stdinit()
+        if mode == 0:
+            pc.psolve(h.tstop)
+        elif mode == 1:
+            while h.t < h.tstop:
+                pc.psolve(h.t + 1.0)
+        else:
+            while h.t < h.tstop:
+                h.continuerun(h.t + 0.5)
+                pc.psolve(h.t + 0.5)
+        tran = [h.t, h.soma(0.5).v, h.soma(0.5).hh.m]
+
+        assert tv.eq(tvstd)
+        assert (
+            v.cl().sub(vstd).abs().max() < 1e-10
+        )  # usually v == vstd, some compilers might give slightly different results
+        # This check is disabled on GPU because fast imem is not implemented on
+        # GPU: https://github.com/BlueBrain/CoreNeuron/issues/197
+        assert coreneuron.gpu or i_mem.cl().sub(i_memstd).abs().max() < 1e-10
```

Comment:
```suggestion
        assert i_mem.cl().sub(i_memstd).abs().max() < 1e-10
```
looks like a bad merge of `master` after https://github.com/neuronsimulator/nrn/pull/1313